drivers/winc1500/winc1500.c

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (C) 2017 Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_winc1500
+ * @{
+ *
+ * @file
+ * @brief       The WINC1500 driver (Wi-Fi connectivity only)
+ *
+ * @author      Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * @}
+ */
+
+/* Required for strnlen in string.h, when building with -std=c99 */
+#define _DEFAULT_SOURCE 1
+#include <string.h>
+/* From pkg/winc1500 */
+#include "driver/include/m2m_wifi.h"
+#include "driver/source/m2m_hif.h"
+
+#include "winc1500.h"
+#include "winc1500_internal.h"
+#include "winc1500_netdev.h"
+
+#include "xtimer.h"
+#include "log.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define START_HANDLER (1)
+#define STOP_HANDLER  (2)
+
+winc1500_t winc1500 = { .state = 0 };
```

Comment:
I don't think you should have a static initializer here. See my previous comment.

Reply:
Right. I will fix this after some test.